sunscreen_runtime/src/debugger/server.rs

```diff
@@ -117,10 +117,20 @@ async fn get_code(session: web::Path<String>) -> impl Responder {
     let sessions = get_sessions().lock().unwrap();
 
     if sessions.contains_key(session.as_str()) {
-        let curr_session = sessions.get(session.as_str()).unwrap().unwrap_bfv_session();
-        let code_string = &curr_session.source_code;
+        let curr_session = sessions.get(session.as_str()).unwrap();
+
+        let program_string = match curr_session {
+            Session::BfvSession(_) => {
+                serde_json::to_string_pretty(&curr_session.unwrap_bfv_session().source_code)
```

Comment:
This looks kind of weird. Couldn't you just get the session from the match? Something like `BfvSession(s) => serde_json::to_string_pretty(s.source_code)`